parser/src/main/grammar/AyaPsiParser.bnf

```diff
@@ -276,15 +286,15 @@ lambdaExpr ::= KW_LAMBDA lambdaTele+ (IMPLIES expr)?
 matchExpr ::= KW_MATCH exprList clauses
 doExpr ::= KW_DO LBRACE? doBlock RBRACE?
 thisExpr ::= KW_THIS (AT qualifiedId)?
-idiomExpr ::= LIDIOM idiomBlock? RIDIOM { pin=1 }
+idiomAtom ::= LIDIOM idiomBlock RIDIOM { pin=1 }
 pathExpr ::= LPATH pathTele+ RPATH expr partialExpr?
```

Comment:
I guess idioms can be empty as long as there exists an `empty` of that `Alternative`

Reply:
I will do it in a new PR